t/plugin/plugin.t

```diff
@@ -28,6 +28,7 @@ add_block_preprocessor(sub {
     $block;
 });
 
+repeat_each(1);
```

Comment:
https://github.com/apache/apisix/blob/e523132f35fe2eb7e1d06990d037f18f89c9dcba/t/APISIX.pm#L23
We already set it by default?